.github/workflows/promptflow-executor-unit-test.yml

```diff
@@ -74,6 +74,7 @@ jobs:
       run: |
         gci env:* | sort-object name
         az account show
+        pip install langchain
```

Comment:
`requirements-dev.txt` is a general file, I think it would be better to limit the scope of the langchain dependence to only those necessary for running the unit tests.